### PR TITLE
Folder Added: Assignment 6 & 9

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

File: akash1/JavaScript/ASSIn6/ass6.3.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <script>
+            function sum(student) {
+                let sum = 0;
+                for (let i = 0; i < student.length; i++) sum += student[i];
+                return sum;
+            }
+        </script>
+    </head>
+    <body style="background-color: cadetblue">
+
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn6/ass6.3.html]: remove extra line

File: akash1/JavaScript/ASSIn6/ass6.5.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            }
+            function factorial(n) {
+                if (n == 0) return 1;
+                return n * factorial(n - 1);
+            }
+            p.innerHTML += `<div>Enter Number: ${arr}</div><br>`;
+            p.innerHTML += `<div> Factorials in desc: ${factorials.sort(function (a, b) {
+                return b - a;
+            })}
+            </div>`;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn6/ass6.5.html:35]: I think,  this closing div tag should be close before the closing curly breses.

File: akash1/JavaScript/ASSIn9/assn9.1.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+        integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+        crossorigin="anonymous"
+    />
+        <style>
+            .container{
+                width: 46%;
+                border-radius: 5px;
+                border: 2px solid white;
+
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.1.html]: remove extra line.

File: akash1/JavaScript/ASSIn9/assn9.2.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            }
+        .container{
+            box-shadow: 0 10px 12px 0 rgba(213, 215, 216, 0.973), 0 6px 20px 0 rgba(206, 14, 14, 0.918);
+             top: 149px;
+             position: relative;
+            border-top-left-radius: 10px;
+            border-top-right-radius:10px ;
+            width: 40%;
+        }    
+            </style>
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.2.html]: give indentation properly.

File: akash1/JavaScript/ASSIn9/assn9.2.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                    } else {
+                        
+                        document.getElementById("p").innerHTML = " Not Prime Palindrome ";
+                    }
+                }  
+                var string = [], splitStr, revArrChar,revString, i, count=0;
+                string = document.getElementById("input1").value;
+                str_palindrome(string);                        
+            } 
+		</script>
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.2.html:49]: give indentation properly.

File: akash1/JavaScript/ASSIn9/assn9.4.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            var stringArrayRev= stringArray.reverse();
+            var revString=stringArrayRev.join('');
+            if(string==revString){
+               document.getElementById("p").innerHTML="Is Palinmdrome"
+            }
+            else{
+               document.getElementById("p").innerHTML="Is not Palinmdrome"
+			}
+        }
+		</script>
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.4.html:40]: give indentation properly.

File: akash1/JavaScript/ASSIn9/assn9.5.html [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+                    <div class="col">
+                        Enter string:<input type="text" class="form-control" placeholder="" id="input1"><br>   
+                        <button onclick="double_letter()" type="button" class="btn btn-primary align-content-lg-center">Show</button>
+                    </div>
+                </div>
+            <br>
+            <h3 id="p">Double letter: </h3>
+            <p id="pid"></p>
+            <p id="pid1"></p>
+            <br>
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.5.html:59]: give indentation properly

File: akash1/JavaScript/ASSIn9/assn9.6.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+                let now=document.getElementById('input1').value;
+                let tut=now.match( /\w+/g ).length;  
+                var spaceCount = (now.split(" ").length - 1); 
+                document.getElementById('p').innerHTML+="The total words: "+tut+"<br>";  
+                document.getElementById('p').innerHTML+="Total whitespaces: "+spaceCount;  
+            }
+		</script>
+	</head>
+	<body>
+		<div class="container text-white border-primary bg-dark">
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.6.html:38]: give indentation properly.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: akash1/JavaScript/ASSIn9/assn9.1.html [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+				if(final==temp)
+				{
+					document.getElementById("p").innerHTML+="Number is palindrome ";
+				}
+				else
+				{
+					document.getElementById("p").innerHTML+="Number is not palindrome "
+				}
+			}
+		</script>
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn9/assn9.1.html:44]: Give indentation properly

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: akash1/JavaScript/ASSIn6/ass6.5.html (hunk 2 above)
Comment (akash19972306) [reply akash1/JavaScript/ASSIn6/ass6.5.html:35]: we using function in div block

**Review by Pranjali1999 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**